lib/src/command/deps.dart

```diff
@@ -254,7 +251,7 @@ class DepsCommand extends PubCommand {
     // The work list for the breadth-first traversal. It contains the package
     // being added to the tree, and the parent map that will receive that
     // package.
-    var toWalk = Queue<Pair<Package, Map<String, Map>>>();
+    var toWalk = Queue<Pair<Package?, Map<String, Map>>>();
```

Comment:
Are we sure the `Package` can be null? It seems we immediately call `!` on it when taking it out.